File: Saving.cpp
#include <iostream>
using std::cout;
using std::endl;
using std::cin;

#include <string>
using std::string;

#include <fstream>


class Saving: public Account {
    private:
        const double interest = 2/100;
        const double feeAdult = 20.99;
        const double feeSenior = 10.99;
        const int numberOfTransaction = 3;




    public:
        Saving(int age) { 
            int fee = (age >= 65) ? feeSenior : feeAdult;

        }
};

File: Saving.java
package Frontend;

public class Saving extends Account {
    private final int numberOfTransaction = 3;
    private double balance = 0;
	private final double interest = 3/100;
	private String username;
	private String password;

	
	
	
	public Saving(String username, String password) {
		super(username, password);
	}
	
	public void add(double amount) {
		balance += amount;
	}
	
	
}
